pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -243,10 +248,62 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s $out/bin/qemu-system-${stdenv.hostPlatform.qemuArch} $out/bin/qemu-kvm
   '';
 
+  # Normal qemu package does not have installcheck make target.
+  # toolsOnly has custom check.
+  doInstallCheck = toolsOnly;
+
+  installCheckPhase =
+    if toolsOnly
+    then ''
```

Comment:
lib.optionalString

Reply:
`null` avoided a rebuild of `qemu`, but I guess `optionalString` (`""`) is more idiomatic.